viofs/svc/virtiofs.cpp

```diff
@@ -117,6 +117,7 @@ struct VIRTFS
     // Used to handle device remove notification.
     VIRTFS_NOTIFICATION DevHandleNotification{};
 
+    bool CaseInsensitive{ false };
```

Comment:
@viktor-prutyanov I am wondering if it should be the opposite. Case insensitivity as default.